### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: book/security.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ sections:
 
 2) Denying access to your app (*authorization*);
 
-3) Fetching the current User object
+3) Fetching the current User object.
```
Comment (WouterJ) [inline book/security.rst]: can you please update this list to use `#.` instead of `n)`?

File: book/security.rst [hunk 2]
```diff
@@ -14,11 +14,11 @@ is both flexible and (hopefully) fun to work with.
...
 sections:
 
-1) Initial ``security.yml`` setup (*authentication*);
+1. Initial ``security.yml`` setup (*authentication*);
 
-2) Denying access to your app (*authorization*);
+2. Denying access to your app (*authorization*);
 
-3) Fetching the current User object
+3. Fetching the current User object.
```
Comment (xabbuh) [inline book/security.rst]: You can actually really  use the `#` character which will then automatically expand to the right numbers.

Comment (ifdattic) [inline book/security.rst]: Thanks, will keep it in mind :)
I thought the same, but decide that I don't feel like experimenting so will leave it like it is.
Should I change the numbers to `#` or leave it like it is?

Comment (WouterJ) [inline book/security.rst]: yes, please change them to use `#.`

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Ah, all easy wins! So @ifdattic, I guess a congratulations is in order about the Symfony exam? :)

**Comment by ifdattic** [comment]
Yes, thank you @weaverryan :) Published an article about my experience & a few tips for anyone who is thinking about the exam a few days ago. Now got hooked on the anything BDD related topic :D

**Pull request merged**